code/modules/paperwork/pen.dm

```diff
@@ -400,3 +400,44 @@
 	. = ..()
 	icon_state = "[initial(icon_state)][HAS_TRAIT(src, TRAIT_TRANSFORM_ACTIVE) ? "_out" : null]"
 	inhand_icon_state = initial(inhand_icon_state) //since transforming component switches the icon.
+
+//The Security holopen
+/obj/item/pen/red/security
+	name = "security pen"
+	desc = "This is a red ink pen exclusively provided to members of the Security Department. Its opposite end features a built-in holographic projector designed for issuing arrest prompts to individuals."
+	icon_state = "pen_sec"
+	var/holo_cooldown = 0
```

Comment:
Can we use the `COOLDOWN_DECLARE` / `COOLDOWN_START` / `COOLDOWN_FINISHED` macros for this?
Would be nice as a slightly code improvement on the medical penlight too but you didn't write that one.